<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SundayBetweenToolUI</class>
 <widget class="QMainWindow" name="SundayBetweenToolUI">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>365</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>252</width>
    <height>365</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Sunday |Â Between Tool</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(100, 100, 100);

font: 12px &quot;Lucida Grande&quot;;</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>10</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);

font: 18px &quot;Verdana&quot;;</string>
         </property>
         <property name="text">
          <string>BETWEEN TOOL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(68, 68, 68);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="SundayControllerToolBetweenSourceToTarget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string>Source to Target(s)</string>
         </property>
         <property name="+command" stdset="0">
          <string>&quot;SundayBetweenToolPy.SundayBetweenToolBuild('transform')&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SundayControllerToolBetweenSourceToTarget_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string>Joint</string>
         </property>
         <property name="+command" stdset="0">
          <string>&quot;SundayBetweenToolPy.SundayBetweenToolBuild('joint')&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SundayControllerToolBetweenDownstream">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string>Downstream (Joint)</string>
         </property>
         <property name="+command" stdset="0">
          <string>&quot;SundayBetweenToolPy.SundayBetweenToolBuildJoint('downstream')&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SundayControllerToolBetweenEntireHieraracy">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string>Entire Hierarchy (Joint)</string>
         </property>
         <property name="+command" stdset="0">
          <string>&quot;SundayBetweenToolPy.SundayBetweenToolBuildJoint('hierarachy')&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string> OPTIONS </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Between Object</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="SundayControllerToolBetweenBuildType">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>193</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>193</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimumContentsLength">
             <number>22</number>
            </property>
            <property name="duplicatesEnabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Box</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cylinder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Helix (Spring)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Last Selected Object</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="SundayControllerToolBetweenDynamicScaleCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="text">
             <string>Dynamic Scale</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="SundayControllerToolBetweenSelectObjectsCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="text">
             <string>Select Between Object(s)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>26</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Width Scale</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="SundayControllerToolBetweenWidthScale">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>1.0</string>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionHelp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDraft">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Draft</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
  <action name="actionPreview_With_Indirect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Preview With Indirect</string>
   </property>
  </action>
  <action name="actionProduction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Production Film</string>
   </property>
  </action>
  <action name="actionProduction_Still">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Production Still</string>
   </property>
  </action>
  <action name="actionSave_Setting">
   <property name="text">
    <string>Save Setting</string>
   </property>
  </action>
  <action name="actionSave_Preset_x">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Preset</string>
   </property>
  </action>
  <action name="actionModelling">
   <property name="text">
    <string>Concept</string>
   </property>
  </action>
  <action name="actionRendering">
   <property name="text">
    <string>Modeling</string>
   </property>
  </action>
  <action name="actionAniamtion">
   <property name="text">
    <string>Shading</string>
   </property>
  </action>
  <action name="actionAnimation">
   <property name="text">
    <string>Animation</string>
   </property>
  </action>
  <action name="actionRendering_2">
   <property name="text">
    <string>Rendering</string>
   </property>
  </action>
  <action name="actionPipelienRendering">
   <property name="text">
    <string>Pipeline Rendering</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
